.github/workflows/checks.yaml

```diff
@@ -366,9 +366,14 @@ jobs:
         uses: taiki-e/install-action@56ab7930c591507f833cbaed864d201386d518a8
         with:
           tool: cargo-nextest
-      - name: Test public C api with NULL arguments
-        run: "cargo +nightly miri nextest run -j4 -p test-libbzip2-rs-sys null::"
-        env:
-          RUSTFLAGS: "-Ctarget-feature=+avx2"
-      - name: Test allocator with miri
-        run: "cargo +nightly miri nextest run -j4 -p libbzip2-rs-sys allocate::"
+      - name: Test public C api with miri
+        run: |
+          cargo +nightly miri nextest run -j4 -p test-libbzip2-rs-sys -- \
+            --skip buff_to_buff_ --skip bzip2_testfiles::commons_compress_zip64support \
+            --skip bzip2_testfiles::dotnetzip_ --skip bzip2_testfiles::go \
+            --skip bzip2_testfiles::lbzip2_ch255 --skip bzip2_testfiles::lbzip2_fib \
+            --skip bzip2_testfiles::lbzip2_idx899999 --skip bzip2_testfiles::lbzip2_repet \
+            --skip bzip2_testfiles::pyflate_45MB_ --skip chunked:: --skip compress_sample1 \
+            --skip compress_sample2 --skip decompress_sample1 --skip decompress_sample2
```

Comment:
this is kind of fragile when we add a new test. I suspect we can have a specific `miri::` module for the miri tests, and just accept that we don't run the rest on CI

Reply:
Something like this?

Reply:
yes that should work for now